nomad/structs/network.go

```diff
@@ -74,6 +76,58 @@ func (idx *NetworkIndex) getUsedPortsFor(ip string) Bitmap {
 	return used
 }
 
+func (idx *NetworkIndex) Copy() *NetworkIndex {
+	c := new(NetworkIndex)
+	*c = *idx
+
+	c.AvailNetworks = copyNetworkResources(idx.AvailNetworks)
+	c.NodeNetworks = copyNodeNetworks(idx.NodeNetworks)
+	c.AvailBandwidth = helper.CopyMapStringInt(idx.AvailBandwidth)
+	c.UsedBandwidth = helper.CopyMapStringInt(idx.UsedBandwidth)
```

Comment:
No need to change it because I think we're already inconsistent but I try to match the Copy order to the original field order in the struct to make visually checking for missed fields easier.

(We really need to autogenerate Copy methods :sob: )

Reply:
Re-order the field, which made me realized that I forgot the `nil` checks, so big +1 for autogerated code 😄